Library/Formula/znapzend.rb

```diff
@@ -0,0 +1,17 @@
+class Znapzend < Formula
+  desc "zfs backup with remote capabilities and mbuffer integration"
+  homepage "http://www.znapzend.org"
+  url "https://github.com/oetiker/znapzend/releases/download/v0.15.3/znapzend-0.15.3.tar.gz"
+  sha256 "27f7c218f4e71beac7b2ba52386a7742506d45c0b3fd40104f44ccef0f373b6e"
+
+  depends_on "perl" if MacOS.version <= :mavericks
```

Comment:
under "depends on"? brew audit didn't complain (but brew audit, admittedly also didn't complain about comments in the test block)...